neps/nep-0568.md

```diff
@@ -428,27 +474,22 @@ The state sync algorithm defines a `sync_hash` that is used in many parts of the
 
 ## Consequences
 
-```text
-[This section describes the consequences, after applying the decision. All consequences should be summarized here, not just the "positive" ones. Record any concerns raised throughout the NEP discussion.]
-```
-
 ### Positive
 
-* p1
+* The protocol is able to execute resharding even while only a fraction of nodes track the split shard.
+* New resharding can happen in the matter of minutes instead of hours.
```

Comment:
Should we say the resharding happens "instantaneously" from the point of view of NEAR users?

Reply:
Btw, is this section supposed to be filled by us or the NEP reviewers? I've seen it empty for a few NEPs.

Reply:
Well, it happened instantaneously before as well. It's just that we need less "background compute power" now. I clarified it.

For NEP reviewers we have "Benefits" and "Concerns" sections below. It seems very reasonable to provide our initial view at pros and cons before review.